content/en/ecosystem/adopters.md

```diff
@@ -0,0 +1,41 @@
+# prettier-ignore
+spelling: cSpell:ignore Dapr Datenrettungsdienste Globale Logicmonitor Logz Wandera Zocdoc
+# All spelling entries must be on a single line
+---
+
+A non-exhaustive, alphabetized list of organizations that have adopted
+OpenTelemetry.
+
+<!-- prettier-ignore-start -->
+Company                                           | Components                      |  Notes
+------------------------------------------------  | ------------------------------- |  :-----------------------------------------------------------------------------------------------------------------:
+[Amazon Web Services](https://aws.amazon.com/)    | Collector, Java, JS, Python     |  [website](https://aws.amazon.com/otel/)
+[AppDirect](https://www.appdirect.com/)           | Collector                       |
+[Care.com](https://www.care.com)                  | Collector, Go, Java, JS, .NET   |
+[Cloud Scale](https://www.cloudscaleinc.com)      | Collector, Python    |
+[Dynatrace](https://www.dynatrace.com/)           | Collector, Java, JS, Python, Go, .NET |
+[EcoBee](https://www.ecobee.com/)                 | Collector, Java                 |  [blog post](https://www.honeycomb.io/blog/bees-working-together-how-ecobees-engineers-adopted-honeycomb-for-visibility-into-system-optimization-and-customer-experience)
+[F5](https://www.f5.com/)                         | Collector.                      |
+[Google Cloud Platform](https://cloud.google.com) | Collector, Go, Java, JS, Python |
+[Grafana Labs](https://grafana.com/)              | Collector                       |
+[Honeycomb](https://honeycomb.io)                 | Go                              |  [blog post](https://www.honeycomb.io/blog/interview-with-honeycomb-engineer-chris-toshok-dogfooding-opentelemetry/)
+[Instana](https://www.instana.com)                | Collector, JS                                       |  [docs](https://www.ibm.com/docs/en/obi/current?topic=apis-opentelemetry)
+[Jaeger](https://jaegertracing.io)                | Jaeger                          |  [blog post](https://medium.com/jaegertracing/jaeger-embraces-opentelemetry-collector-90a545cbc24)
+[Lightstep](https://lightstep.com)                | Go, JS                          |
+[Logicmonitor](https://www.logicmonitor.com/)     | Collector, Java, Go, JS, Python, .Net |
+[Logz.io](https://logz.io)                        | Collector, Java, C++, Go, JS, K8s         |  [blog](https://logz.io/learn/opentelemetry-guide/)
+[Microsoft](https://www.microsoft.com/)           | Collector, C++, Go, Java, JS, Python, Ruby, .NET    | [Azure blog](https://techcommunity.microsoft.com/t5/azure-monitor/opentelemetry-azure-monitor/ba-p/2737823), [Dapr docs](https://docs.dapr.io/operations/monitoring/tracing/otel-collector/), [GitHub blog](https://github.blog/2021-05-26-why-and-how-github-is-adopting-opentelemetry/)
+[OpenTelemetry](https://opentelemetry.io)         | Collector, JS                   |
```

Comment:
does anyone understand what this line represents?

Reply:
LOL ... that is indeed an interesting entry. Do you want me to pull it out. It's a bit too circular otherwise :).

Reply:
Yeah. I would :-)

Reply:
Dropped ✨

Reply:
rm